### ISSUE TITLE
Unable to fix a fixable violation

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#920]
```sh
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       7.0.2   The Doctrine Coding Standard is a ...
slevomat/coding-standard                       6.1.5   Slevomat Coding Standard for PHP_C...
squizlabs/php_codesniffer                      3.5.4   PHP_CodeSniffer tokenizes PHP, Jav...
```

Consider the following code example:

```php
<?php

declare(strict_types=1);

function foo() : void
{
    if (true) { // Some comment
        return;
    }
}
```

The following violation is reported:
```
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 8 | ERROR | [x] Expected 1 lines before "return", found 0.
   |       |     (SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountBeforeControlStructure)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
An attempt to auto-fix the file fails:
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                              FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------
```
The failure to fix this little one prevents PHPCBF from fixing the rest of the violations in the actual file that contains this violation.

**Comment by kukulich** [comment]
Probably duplicate to https://github.com/slevomat/coding-standard/issues/903 because I'm not able to reproduce it in `master`.

**Comment by morozov** [comment]
`master` worked for me. Thanks, @kukulich.
